nss-day-cohort-41/tabloidmvc-the-gefilte-fish
# Aj post edit

# Description
Checked Functionality of 
-Create Post
-Post Details
Added Edit Functionality.
Added Delete.
Fixes #8 #7 #3 #6 #5 
Please delete options that are not relevant.
# Testing Instructions
Check Edit by selecting a post and changing all input fields.
Delete is a two-part system. Hit the trashcan and you will be redirected.
Once at the followup page verify it is the correct post and confirm the delete.
# Checklist:
